.gitignore

```diff
@@ -15,5 +15,6 @@ hack/tls-setup/certs
 .idea
 /contrib/raftexample/raftexample
 /contrib/raftexample/raftexample-*
+.vendor
```

Comment:
Is the leading dot supposed to be there?

Reply:
Good catch. Done.